src/main/resources/db/changelog/changes/06-add-admin-user.yaml

```diff
@@ -0,0 +1,47 @@
+databaseChangeLog:
+  - changeSet:
+      id: add-admin-user
+      author: ostapDanyliv
+      changes:
+        - insert:
+            tableName: users
+            columns:
+              - column:
+                  name: id
+                  value: 1
+              - column:
+                  name: email
+                  value: 'admin@example.com'
+              - column:
+                  name: password
+                  value: '$2a$10$DpPyUa9hs8yjCvo/kn4KQu1GzlONLCJYgb5xCpborapQ0wGBEobcq'
+              - column:
+                  name: first_name
+                  value: 'Admin'
+              - column:
+                  name: last_name
+                  value: 'ADMINY4'
+        - insert:
+            tableName: users_roles
+            columns:
+              - column:
+                  name: id
+                  value: 1
+              - column:
+                  name: user_id
+                  value: 1
```

Comment:
It's better to use `valueComputed` to avoid potential errors. (Check common mistakes)